miniwin/CMakeLists.txt

```
@@ -44,16 +44,16 @@ if(NOT WINDOWS_STORE)
     message(STATUS "🧩 OpenGL 1.x support not enabled — needs OpenGL")
   endif()
 
-  find_library(OPENGL_ES2_LIBRARY NAMES GLESv2)
-  if(EMSCRIPTEN OR OPENGL_ES2_LIBRARY)
-    message(STATUS "Found OpenGL: enabling OpenGL ES 2.x renderer")
-    target_sources(miniwin PRIVATE src/d3drm/backends/opengles2/renderer.cpp)
-    list(APPEND GRAPHICS_BACKENDS USE_OPENGLES2)
-    if(OPENGL_ES2_LIBRARY)
-      target_link_libraries(miniwin PRIVATE ${OPENGL_ES2_LIBRARY})
+  find_library(OPENGL_ES3_LIBRARY NAMES GLESv3)
```

This should fix it not building on Linux, I know it seems a bit odd ...
```suggestion
  find_library(OPENGL_ES3_LIBRARY NAMES GLESv2)
```